dabest/_classes.py

```diff
@@ -658,6 +659,16 @@ def __init__(self, control, test, effect_size,
                 # in terms of rank (eg. all zeros.)
                 pass
 
+            # Likelihood Q-Ratio test:
+            # Assumes a gross-error model of distributions
+            try:
+                lqrt_ratio = lqrt.lqrtest_rel(control, test)
```

Comment:
Hi @adam2392 

The PR failed automatic Travis CI checks; it seems you used  `lqrt.lqrtest_rel()` for both paired and unpaired cases. This creates an error when the control and test arrays have different lengths.

Could you include a check where `lqrt.lqrtest_ind()` is used if `is_paired` is False?

Reply:
@josesho done!